lib/Backends/CPU/Transforms.cpp

```diff
@@ -34,11 +34,12 @@ using llvm::isa;
 static Node *optimizeCPUConv(ConvolutionNode *CN, Function *F) {
   auto depth = CN->getFilter().dims()[0];
   auto *M = F->getParent();
+  auto group = CN->getGroup();
 
-  // The depth dimension must be a multiple of 64 to perform the
+  // The depth dimension in each group must be a multiple of 64 to perform the
   // transformation. This transformation is currently only profitable on
   // low-channel convolutions.
-  if (depth < 64 || (depth % 64) != 0) {
+  if (depth < 64 || ((depth / group) % 64) != 0) {
```

Comment:
@beicy Can you extract this expression into a new line and add a comment and perhaps a variable names? 

// Make sure that the depth group is divisible by 64. 
bool divisibleBy64 = ((depth / group) % 64) != 0;